README.md

```diff
@@ -23,27 +23,16 @@ Though most configuration sources found online highly recommend running HA on a
     - Windows system was [here](https://www.youtube.com/watch?v=dp-0hVjEo6A). 
     - Mac ---> ToDo [here]. 
     - Linux [here](https://github.com/brandondombrowsky/BastCastle/wiki/Wireguard-VPN-Setup#install-docker) 
-2. HA will scan for and find most of your devices. ---> ToDo check this
+2. HA will scan for and find most of your devices. For those devices it doesn't find, use the search tool to locate.
```

Comment:
Looks good. Maybe add a todo we could add a demo of how to do this? It might be so simple that it's not necessary though, just depends on our project end users.

Reply:
Couldn't hurt